### PR TITLE
EZP-29302: UDW displays N/A instead of item's Content Type

### PR DESCRIPTION
**Pull request by tischsoic**
https://jira.ez.no/browse/EZP-29302

### CODE REVIEW

File: src/modules/universal-discovery/universal.discovery.module.js [hunk 1]
```diff
@@ -169,13 +170,16 @@ export default class UniversalDiscoveryModule extends Component {
         this.props.onConfirm(this.addContentTypeInfo([location]));
     }
 
-    addContentTypeInfo(content) {
-        return content.map((item) => {
-            item = JSON.parse(JSON.stringify(item));
+    addContentTypeInfo(contents) {
```
Comment (sunpietro) [inline src/modules/universal-discovery/universal.discovery.module.js]: Why did you change `contents` to an `array`?

File: src/modules/universal-discovery/universal.discovery.module.js [hunk 2]
```diff
@@ -188,10 +192,11 @@ export default class UniversalDiscoveryModule extends Component {
 
     onItemSelect(contentMeta) {
         const isLocationAllowed = !this.props.cotfAllowedLocations.length || this.props.cotfAllowedLocations.includes(contentMeta.id);
+        const contentMetaWitchContentTypeInfo = this.addContentTypeInfo([contentMeta])[0];
```
Comment (sunpietro) [inline src/modules/universal-discovery/universal.discovery.module.js]: Because of your change you're doing a kind of strange thing here. Currently, I don't see any value in changing the way of handling from a single item to an array of items in the `addContentTypeInfo` method

File: src/modules/universal-discovery/universal.discovery.module.js (hunk 1 above)
Comment (sunpietro) [inline src/modules/universal-discovery/universal.discovery.module.js]: `content` is uncountable. Maybe you should use `contentMeta`

File: src/modules/universal-discovery/universal.discovery.module.js (hunk 2 above)
Comment (dew326) [inline src/modules/universal-discovery/universal.discovery.module.js]: `contentMetaWithContentTypeInfo` - you have `witCh`

Comment (sunpietro) [reply src/modules/universal-discovery/universal.discovery.module.js]: Let's call the witch hunter ;)

File: src/modules/universal-discovery/universal.discovery.module.js (hunk 1 above)
Comment (dew326) [reply src/modules/universal-discovery/universal.discovery.module.js]: I would use generic `items`

**Review by dew326 — APPROVED** [review]

**Review by sunpietro — APPROVED** [review]

**Review by micszo — APPROVED** [review]
👍
